packages/mdc-snackbar/foundation.js

```diff
@@ -108,10 +108,12 @@ export default class MDCSnackbarFoundation extends MDCFoundation {
     this.adapter_.addClass(ACTIVE);
     this.adapter_.unsetAriaHidden();
 
-    setTimeout(this.cleanup_.bind(this), data.timeout || MESSAGE_TIMEOUT);
+    this.timeout_id_ = setTimeout(this.cleanup_.bind(this), data.timeout || MESSAGE_TIMEOUT);
```

Comment:
I don't know why our linter didn't catch this, but this variable should be named `timeoutId_` (camelCase with trailing underscore).